src/main/java/sggw/wzim/czasnawypad/service/AttractionRatingService.java

```diff
@@ -0,0 +1,80 @@
+package sggw.wzim.czasnawypad.service;
+
+import jakarta.servlet.http.HttpServletRequest;
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.security.oauth2.server.resource.InvalidBearerTokenException;
+import org.springframework.stereotype.Service;
+import sggw.wzim.czasnawypad.db.AttractionRepository;
+import sggw.wzim.czasnawypad.db.UserRepository;
+import sggw.wzim.czasnawypad.db.dto.AttractionRatingDTO;
+import sggw.wzim.czasnawypad.db.dto.CreateAttractionRatingDTO;
+import sggw.wzim.czasnawypad.db.entity.Attraction;
+import sggw.wzim.czasnawypad.db.entity.AttractionRating;
+import sggw.wzim.czasnawypad.db.entity.User;
+import sggw.wzim.czasnawypad.exception.ApplicationExceptions;
+import sggw.wzim.czasnawypad.mapper.AttractionMapper;
+import sggw.wzim.czasnawypad.mapper.AttractionRatingMapper;
+import sggw.wzim.czasnawypad.db.AttractionRatingRepository;
+
+import java.util.Date;
```

Comment:
Don't use `java.util.Date` as it's deprecated. https://www.baeldung.com/java-convert-localdate-sql-date